doc/alertmanager-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,310 @@
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+
+
+{
+  grafanaDashboards+:: {
+    local clusterTemplate =
+      template.new(
+        name='cluster',
+        datasource='$datasource',
+        query='label_values(alertmanager_alerts, %s)' % $._config.clusterLabel,
+        current='',
+        hide=if $._config.showMultiCluster then '' else '2',
+        refresh=2,
+        includeAll=false,
+        sort=1
+      ),
+
+    'alertmanager-overview.json':
+
+      local alerts =
+        graphPanel.new(
+          'Alerts',
+          datasource='$datasource',
+          span=6,
+          format='none',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('alertmanager_alerts{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}' % $._config, legendFormat='{{ state }}'));
+
+      local alertsRate =
+        graphPanel.new(
+          'Alerts receive rate',
+          datasource='$datasource',
+          span=6,
+          format='ops',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('sum(rate(alertmanager_alerts_received_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m]))' % $._config, legendFormat='Received'))
+        .addTarget(prometheus.target('sum(rate(alertmanager_alerts_invalid_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m]))' % $._config, legendFormat='Invalid'));
+
+      local notifications =
+        graphPanel.new(
+          'Rate',
+          datasource='$datasource',
+          span=4,
+          format='ops',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target('sum(rate(alertmanager_notifications_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])) by (integration)' % $._config, legendFormat='{{integration}}'));
+
+      local notificationErrors =
+        graphPanel.new(
+          'Error',
+          datasource='$datasource',
+          span=4,
+          format='ops',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target('sum(rate(alertmanager_notifications_failed_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])) by (integration)' % $._config, legendFormat='{{integration}}'));
+
+      local notificationDuration =
+        graphPanel.new(
+          'Duration',
+          datasource='$datasource',
+          span=4,
+          format='s',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target(
+          |||
+            histogram_quantile(0.99,
+              rate(alertmanager_notification_latency_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
```

Comment:
So this _is_ actually broken up per instance, isn't it? But then it should also be in the legend.

And speaking about the "per instance" legend: This is actually where the K8s cluster label (usually "cluster") comes into the game again.

For example, here at Grafana Labs, we call our global AM cluster `global-alertmanager/alertmanager`, and this name is used as the job name, in each of the two K8s clusters where we run two instances each, all participating in the global AM cluster. The instances are then called `alertmanager-0` and `alertmanager-1` in _each_ K8s cluster, so the legend needed to include both the `instance` and the `cluster` label, so that it reads "alertmanager-0 in us-east-3" etc.